client/my-sites/earn/main.tsx

```diff
@@ -256,7 +260,15 @@ const EarningsMain = ( { section, query, path }: EarningsMainProps ) => {
 							'Explore tools to earn money with your site. {{learnMoreLink}}Learn more{{/learnMoreLink}}.',
 							{
 								components: {
-									learnMoreLink: <InlineSupportLink supportContext="earn" showIcon={ false } />,
+									learnMoreLink: isJetpackNotAtomic ? (
```

Comment:
@lupus2k Should we show the Jetpack link to Atomic classic view?

Reply:
We decided to keep WPcom links for Atomic sites here: https://github.com/Automattic/wp-calypso/pull/88778#issuecomment-2024819919